manifests/repo.pp

```diff
@@ -23,18 +23,6 @@
     notify   => Anchor['pulpcore::repo'],
   }
 
-  # Only EL8 has DNF modules
-  if $dist_tag == 'el8' {
-    package { 'pulpcore-dnf-module':
-      ensure      => $dist_tag,
```

Comment:
Upgrade wise: I wonder what happens. I suspect the module will remain enabled, even if it doesn't exist anymore. Though we pretty much only use this class for CI, it will be relevant for when we consider real installations. Perhaps something the installer should do via a hook, perhaps something for foreman_maintain.

Reply:
It's looking like  the pulpcore module will need to be disabled -- without it the new pulpcore packages aren't being picked up for update by dnf.

Reply:
Actually I can't confirm this yet -- seems even without the pulpcore module `dnf` isn't picking up the new pulpcore packages. Potentially a packaging issue.

Reply:
After a chat, we decided that we'll be adding the module removal step to our documentation. It will indeed need to be disabled. Furthermore, adding a removal step to the foreman-installer is too late because users already `dnf update` before running the installer.